### ISSUE TITLE
Markdown tables are not recognized

### ISSUE THREAD

**Issue by leodr** [shuding/nextra#127]
I tried to copy the exact code you use [here](https://nextra.vercel.app/features/mdx#table), however, it doesn't understand that it is a table:

![image](https://user-images.githubusercontent.com/39763575/113876403-021a8080-97b8-11eb-8c8a-66be2543a737.png)

The code for that looks like this:

```md
> To achieve faster (re-)build times during development this command only
> transpiles your code to work with the latest versions of Chrome, Safari and
> Firefox.

| Syntax        | Description | Test Text     |
| :---          |    :----:   |          ---: |
| Header        | Title       | Here's this   |
| Paragraph     | Text        | And more      |
| Strikethrough |             | ~~Text~~      |

## Parameters
```

Using Next 10.1.3, nextra 0.4.4 and nextra-theme-docs 1.1.6

**Comment by brandonpittman** [comment]
Task lists also aren't being formatted correctly (and are missing the `contains-task-list` class on the generated `<ul>`. I'd imagine it's connected to whatever is causing this issue as well.

**Comment by RyanTheAllmighty** [comment]
I've looked into this myself, and it seems to be related to changes in `@mdx-js/loader`. I also had issues with the `MDXProvider` in `nextra-theme-docs` using an old version of `@mdx-js/react` which caused the custom elements for say headers, link, code blocks, etc, not working.

Seems the dependency locking in some of the packages are bringing in version `2.0.0-next.9` of the `@mdx` related packages which causes some issues in the dependency chain.

Either way I managed to fix it by:

- Locking `@mdx-js/loader` in `package.json` to version `2.0.0-next.8`: `"@mdx-js/loader": "2.0.0-next.8",`
- Removing the `dependencies` object in my `package-lock.json` for `nextra-theme-docs` since it's explicitely looking for version `^1.6.16`, so removing that dependencies object will force npm to simply use the default correct one

YMMV of course, but it worked for me
